flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/BuiltInMethods.scala

```diff
@@ -118,6 +118,13 @@ object BuiltInMethods {
     Types.lookupMethod(
       classOf[ScalarFunctions], "concat_ws", classOf[String], classOf[Array[String]])
 
+  val REPLACE = Types.lookupMethod(
```

Comment:
The `REPLACE` method has been provided in `org.apache.calcite.util.BuiltInMethod`. There's no need to duplicate it here.

Reply:
The same reason like the above. The calcite builtin `REPLACE` can not handle null parameters, which may throw NPE.